src/Microsoft.Extensions.FileProviders.Physical/project.json

```diff
@@ -38,9 +38,11 @@
     "netstandard1.3": {
       "dependencies": {
         "System.Collections.Concurrent": "4.0.12-*",
+        "System.Diagnostics.Debug": "4.0.11",
```

Comment:
This is going away: https://github.com/aspnet/FileSystem/commit/0bf9850d71ddf91473052ecff9c9087326f01fa8